cost/aws/instance_cloudwatch_utilization/AWS Inefficient Instance Utilization using CloudWatch.pt

```diff
@@ -1,10 +1,17 @@
-name "AWS Instance CloudWatch Utilization"
+name "AWS Inefficient Instance Utilization using CloudWatch"
 rs_pt_ver 20180301
 type "policy"
-short_description "Gathers AWS CloudWatch data for instances on 30 day intervals. See the [README](https://github.com/rightscale/policy_templates/tree/master/cost/aws/instance_cloudwatch_utilization/) and [docs.rightscale.com/policies](http://docs.rightscale.com/policies/) to learn more."
-long_description "Version: 1.5"
+short_description "This Policy Template gathers AWS instances with inefficient utilization using CloudWatch CPU and Memory Metrics over a 30 day average and downsized after approval. See the [README](https://github.com/rightscale/policy_templates/tree/master/cost/aws/instance_cloudwatch_utilization/) and [docs.rightscale.com/policies](http://docs.rightscale.com/policies/) to learn more."
+long_description ""
 severity "low"
 category "Cost"
+tenancy "single"
+info(
+  version: "2.0",
+  provider: "AWS",
+  service: "EC2",
+  policy_set: ""
```

Comment:
please use this value for the policy_set: 
**Idle Compute Instances**
--

Reply:
I have updated the policy_set "Inefficient Instance Usage" value from the spreadsheet.

Reply:
yes, thank you.  I copied the wrong value.